docs/benchmarks/corfu-table.md

```diff
@@ -0,0 +1,73 @@
+
+# Large Corfu Table benchmark
+
+This set of micro benchmarks compares EhCache vs RocksDb performance characteristics.
+
+### The Setup
+Mac Book Pro 2017
+ - Processor 2.8 GHz Intel Core i7
+ - 16 GB 2133 MHz LPDDR3
+ - SSD
+
+#### RocksDb parameters:
+ - disable WAL
+ - increase WriteBufferSize to 256 mb
+ - increase MaxWriteBufferNumber from 2 to 4
+
+#### EhCache parameters:
+ - max heap entries 100.000
+ - max disk quota (mb) 50.000
+
+## PUT
+#### Single thread `Put` method. 
+
+![Put. Single Thread performance benchmark](img/single-thread-put.jpg)
+
+
+#### Four threads `Put` method.
+ 
+![Put. Four Threads performance benchmark](img/four-threads-put.jpg)
+
+## GET
+#### Single thread `Get` method. 
+
+![Get. Single Thread performance benchmark, 1 million records](img/single-thread-get-1-mil.jpg)
+
+
+#### Four threads `Get` method.
+ 
+![Get. Four Threads performance benchmark, 1 million records](img/four-threads-get-1-mil.jpg)
+
+EhCache can't manage 10 million records data set, it throws GC error.
+
+RocksDb 50-100% faster than EhCache for single threaded application.
+RocksDb provides better performance for multithreaded applications.
+
+Conclusion:
+  - RocksDb more stable and solid solution then EhCache 
```

Comment:
Would be necessary to compare the resource consumptions of the two engines compared for a fair comparison before making conclusions right?
(RocksDb is not designed or optimized to be an ephemeral datastore - it is designed to be a fast persistent datastore.)

Reply:
I compare persistency features, so according to the numbers, it works better for us in this particular case.
Agree with you, maybe my statement is too strong.
Any ideas about how the sentence can be modified?

Reply:
Maybe you can just qualify it saying "RocksDb is more performant than EhCache" ?

Reply:
done